ui/app/AppLayouts/Profile/views/wallet/NetworksView.qml

```diff
@@ -0,0 +1,48 @@
+import QtQuick 2.13
+
+import shared.status 1.0
+
+import "../../stores"
+import utils 1.0
+
+Column {
```

Comment:
again as discussed we will improve this UI at a later point as the UI may change anyway.